pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -779,6 +781,15 @@ private Object applyChildPicklistAnswerPredicate(InterpreterContext ictx, Predic
                     .flatMap(child -> ((PicklistAnswer) child).getValue().stream())
                     .map(SelectedPicklistOption::getStableId)
                     .anyMatch(optionStableId::equals);
+        } else if (predicateCtx instanceof HasOptionStartsWithPredicateContext) {
```

Comment:
Should we add support for regular picklist as well instead of just child picklists? Having one and not the other might cause confusions. I suggest adding same support in `applyPicklistAnswerPredicate()`.

Reply:
Added support in `applyPicklistAnswerPredicate()`.